script/lint

```diff
@@ -15,7 +15,7 @@ printf "%s\n" $files
 echo "================"
 echo "LINT with flake8"
 echo "================"
-pre-commit run flake8 --files $files
+pre-commit run flake8 --max-complexity 25 --files $files
```

Comment:
This should not be needed, however, maccabe is missing from the `additional_dependencies` in the `.pre-commit-config.yaml` flake8 entry.